File: AD-HOC/2443.py
entrada = list(map(int, input().split()))
a = entrada[0]
b = entrada[1]
c = entrada[2]
d = entrada[3]

denom = b*d
num = (denom/b)*a + (denom/d)*c

for i in range(100, 1, -1):
  if (num%i == 0 and denom%i == 0):
    num = num/i
    denom = denom/i

print(int(num), int(denom))

File: AD-HOC/2443.js
var lines = input.split(' ').map(Number);

let [a, b, c, d] = lines;

let denom = b*d, num = (denom/b)*a + (denom/d)*c;

for (let i = 100; i >= 2; i--) {
  if (num%i == 0 && denom%i == 0) {
    num = num/i;
    denom = denom/i;
  }
}

console.log(num, denom);
